src/elements/play-preview.ts

```diff
@@ -169,4 +170,27 @@ export class PlayPreview extends LitElement {
         this.#meta
       )
   }
+
+  private mockImageUpload = (
+    event: CustomEvent<{
+      blob: Blob
+      uploadRequest: null | Promise<UploadResponse>
+    }>
+  ): void => {
+    event.detail.uploadRequest = new Promise(resolve => {
+      window.setTimeout(() => {
```

Comment:
:no_bell: :nail_care: suggest writing as `setTimeout()` or `globalThis.setTimeout()` so code is default isomorphic.